tasks.py

```diff
@@ -1059,6 +1059,9 @@ def ci_build_and_install_quiche(c, repo):
     c.run(f'echo {quiche_hash}"  "quiche-{quiche_version}.tar.gz | sha256sum -c -')
     c.run(f'tar xf quiche-{quiche_version}.tar.gz')
     with c.cd(f'quiche-{quiche_version}'):
+        # Disable SONAME in the quiche shared library, we do not intend this library to be used by anyone else and it makes things more complicated since we rename it to libdnsdist-quiche
```

Comment:
this duplication (which is not new, of course) is a pity

Reply:
Indeed. I'll ponder making `install_quiche.sh` generic enough to remove the duplication.

Reply:
Done! CI is in progress and I did a packages test at https://github.com/rgacogne/pdns/actions/runs/10485427664